xmtp/src/conversations.rs

```diff
@@ -480,10 +479,9 @@ mod tests {
         let unprocessed_messages = alice_client.store.get_unprocessed_messages().unwrap();
         assert_eq!(unprocessed_messages.len(), 1);
 
-        // TODO replace with Client.refresh_user_installations. Requires us to refactor the SDK so that
-        // two XMTP clients can share the same API client
         alice_client
-            .create_outbound_session(&bob_client.account.contact())
+            .refresh_user_installations(&bob_client.wallet_address())
+            .await
             .unwrap();
```

Comment:
`refresh_user_installations` fetches Bob's contacts from the network before creating a session with them. Before this change, it would have fetched 0 contacts because the API client that Bob published to on initialization was different from the API client that Alice is fetching from here